api-report/shared-object-base.api.md

```diff
@@ -88,11 +84,13 @@ export function serializeHandles(value: any, serializer: IFluidSerializer, bind:
 // @public
 export abstract class SharedObject<TEvent extends ISharedObjectEvents = ISharedObjectEvents> extends SharedObjectCore<TEvent> {
     constructor(id: string, runtime: IFluidDataStoreRuntime, attributes: IChannelAttributes);
+    // (undocumented)
```

Comment:
I have been unable to fix that these are now considered "undocumented". I also do not know how to see and generated documentation produced from this code, nor do the inheritDoc tags work in vs code, so I have no way to test or use the generated documentation.

Reply:
As far as I can tell, this file is the only thing currently consuming the .api.json file.

It's unclear to me if this documentation being omitted here is a good thing or a bad thing:
The entire entry for these methods is omitted from the .api.json file. This could mean that it detected that the base method from the interface is sufficient to describe it, which I think would be a good thing. Alternativity it could mean that I broke the documentation.

Either way, we don't have anything that consumes uses that documentation, and vs-code intelisense is broken due to there being an inherit doc comment in the first place, so that did not regress.

Thus the one thing I use (vs code intelisense) is unchanged (still broken), and the rest is ambiguously better or worse. Thus I guess it should be find to make this change?

Reply:
This is probably because the inheritDoc reference now crosses a package boundary, which doesn't work today: https://github.com/microsoft/rushstack/issues/1195

> If an @inheritDoc tag refers to an external package, API Extractor does not process it. But API Documenter will process it, if it can find the target in one of its .api.json files.

Reply:
> The entire entry for these methods is omitted from the .api.json file. This could mean that it detected that the base method from the interface is sufficient to describe it, which I think would be a good thing.

The API report should not include inherited members, so this is correct.